funcs/func_groupcount.c

```diff
@@ -51,37 +56,315 @@
 			channel's current group if not specified (and non-empty).</para>
 		</description>
 	</function>
+	<function name="GROUP_MATCH_LIST" language="en_US">
+		<synopsis>Find groups by regular expression</synopsis>
+		<syntax argsep="@">
+			<parameter name="group_regex"/>
+			<parameter name="category_regex"/>
+		</syntax>
+		<description>
+			<para>Search for groups matching group@category. This search will look at all known groups and filter by the regex(s) provided.</para>
+			<para>If only the group_regex is specified, then return group@category results that only match the group name against the group_regex</para>
+			<para>If only the category_regex is specified, then return group@category results that only match the category name against the category_regex.</para>
+			<para>If the search regex parameters are entirely empty, all group@category will be returned.</para>
+			<para>Uses standard regular expression matching (see regex(7)).</para>
+			<example>
```

Comment:
The example tag should have a title attribute, move "Find groups containing the string foo" to the title